features/console.yml.dist

```diff
@@ -0,0 +1,187 @@
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "9"
+  #   firefox_android: "9"
+  #   safari: "3"
+  #   safari_ios: "1"
+  - api.console.substitution_strings
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "5"
+  #   firefox_android: "5"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.debug_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "6"
+  #   firefox_android: "6"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.trace_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "8"
+  #   firefox_android: "8"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.dir_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "9"
+  #   firefox_android: "9"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.groupEnd_static
+  - api.console.group_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "10"
+  #   firefox_android: "10"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.timeEnd_static
+  - api.console.time_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "28"
+  #   firefox_android: "28"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.assert_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "30"
+  #   firefox_android: "30"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.count_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "39"
+  #   firefox_android: "39"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.dirxml_static
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "5"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "16"
+  #   firefox_android: "16"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.console.profileEnd_static
```

Comment:
BCD says that `profile` was removed in Firefox in 102. But BCD also says that `profileEnd` didn't get the same treatment, and that seems like a BCD bug to me. I would expect both to either be present or not.
In fact, both are defined for me in Firefox Nightly right now.

Reply:
I see you've removed `profile` but left `profileEnd`. I guess we should instead either have them both in, or out. In this case, I think this is more likely a BCD bug. When I tested quickly, both functions seemed available in Firefox.